api/python/ai/chronon/repo/hub_uploader.py

```
@@ -3,27 +3,33 @@
+
+    # Call Zipline hub with `names_and_hashes` as the argument to get back
+    names_and_hashes = {name: local_conf.hash for name, local_conf in local_repo_entities.items()}
+    changed_entity_names = zipline_hub.call_diff_api(names_and_hashes)
+
     # a list of names for diffed hashes on branch
-    changed_entity_names = local_repo_entities
     return {k: local_repo_entities[k] for k in changed_entity_names}
 
 
+
 def _build_local_repo_hashmap(root_dir: str):
```

Flake8 (7.2.0)

[error] too many blank lines (3)

(E303)